contrib/test_jenkins.sh

```diff
@@ -632,6 +632,29 @@ run_gtest() {
 	fi
 }
 
+#
+# Run the test suite (gtest) in release configuration
+#
+run_gtest_release() {
+
+	echo "1..1" > gtest_release.tap
+
+	../contrib/configure-release --prefix=$ucx_inst --enable-gtest
+	$MAKE clean
+	$MAKE
+
+	export GTEST_SHARD_INDEX=$worker
```

Comment:
in this case, i think need to set INDEX=0. TOTAL_SHARDS=1 so it will always run the test

Reply:
@yosefe done